persistenceagent/rockcraft.yaml

```diff
@@ -0,0 +1,61 @@
+name: kfp-persistence
+summary: Reusable end-to-end ML workflows built using the Kubeflow Pipelines SDK
+description: |
+  This component serves as the backend persistence agent of Kubeflow pipelines.
+version: 2.0.0-ibeta.1_22.04_1 # version format: <KF-upstream-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:22.04
+services:
+  persistenceagent:
+    override: replace
+    summary: "persistenceagent service"
+    startup: enabled
+    user: appuser
```

Comment:
use ubuntu user

Reply:
Using username that was specified in upstream.